api/api.py

```diff
@@ -6,6 +6,7 @@
 cors = CORS(app)
 app.config['CORS_HEADERS'] = 'Content-Type'
 
+
```

Comment:
Why'd you add extra line here and delete one at the end?

Reply:
PEP standards expect 2 blank lines before a decorator and a blank line at the end of the file. My PyCharm IDE highlighted them.

Reply:
PEP standards expect 2 blank lines before a decorator and a blank line at the end of the file. My PyCharm IDE highlighted them.

Reply:
Sounds good